mittab/libs/data_import/import_scratches.py

```diff
@@ -18,40 +18,38 @@
 #OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #THE SOFTWARE.
 
-from tab.models import *
+from mittab.apps.tab.models import *
 import xlrd
 import csv
 from xlwt import Workbook
 
 def import_scratches(fileToImport):
-    wb = xlrd.open_workbook(fileToImport)
-    sh = wb.sheet_by_index(0)
+    sh = xlrd.open_workbook(fileToImport).sheet_by_index(0)
     num_scratches = 0
     found_end = False
+    scratch_errors = []
     while found_end == False:
         try:
             sh.cell(num_scratches,0).value
             num_scratches +=1
         except IndexError:
             found_end = True
-    for i in range(num_scratches):
-        if i != 0:
+    for i in range(1, num_scratches):
+        try:
+            team_name = sh.cell(i,0).value
+            t = Team.objects.get(name = team_name)
+            judge_name = sh.cell(i,1).value
+            j = Judge.objects.get(name = judge_name)
+            s_type = sh.cell(i,2).value.lower()
+            if s_type == "team scratch" or s_type == "team":
+                s_type = 0
+            elif s_type == "tab scratch" or s_type == "tab":
+                s_type = 1
+            s = Scratch(judge = j, team = t, scratch_type = s_type)
             try:
-                team_name = sh.cell(i,0).value
-                t = Team.objects.get(name = team_name)
-                judge_name = sh.cell(i,1).value
-                j = Judge.objects.get(name = judge_name)
-                s_type = sh.cell(i,2).value
-                if s_type == "Team Scratch":
-                    s_type = 0
-                elif s_type == "Tab Scratch":
-                    s_type = 1
-                s = Scratch(judge = j, team = t, scratch_type = s_type)
-                try:
-                    s.save()
-                    print "Created scratch %s, %s" % (j,t)
-                except:
-                    print "Failed to create scratch %s, %s" % (j,t)
+                s.save()
             except:
-                print "Epic fail"
-    print "scratches entered"
+                scratch_errors.append[[j,t]]
+        except:
+            print "Epic fail"
```

Comment:
Heh, maybe except Exception as e and then print it out? Sometimes helps me out when a team needs help if I can read the error log and see what's going on.